tlslite/constants.py

```diff
@@ -592,36 +605,89 @@ class CipherSuite:
+    TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384 = 0xC024
+    ietfNames[0xC024] = 'TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384'
+    TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA256 = 0xC025
+    ietfNames[0xC025] = 'TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA256'
+    TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA384 = 0xC026
+    ietfNames[0xC026] = 'TLS_ECDH_ECDSA_WITH_AES_256_CBC_SHA384'
+
+    # RFC 5289 - ECC Ciphers with SHA-256/SHA-384 HMAC and AES-GCM
     TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256 = 0xC027
     ietfNames[0xC027] = 'TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256'
     TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384 = 0xC028
     ietfNames[0xC028] = 'TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384'
+
+    # RFC 5289 - ECC Ciphers with SHA-256/SHA-384 HMAC and AES-GCM
+    # unsupported
+    TLS_ECDH_RSA_WITH_AES_128_CBC_SHA256 = 0xC029
+    ietfNames[0xC029] = 'TLS_ECDH_RSA_WITH_AES_128_CBC_SHA256'
+    TLS_ECDH_RSA_WITH_AES_256_CBC_SHA384 = 0xC02A
+    ietfNames[0xC02A] = 'TLS_ECDH_RSA_WITH_AES_256_CBC_SHA384'
+    TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256 = 0xC02B
+    ietfNames[0xC02B] = 'TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256'
+    TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384 = 0xC02C
+    ietfNames[0xC02C] = 'TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384'
+    TLS_ECDH_ECDSA_WITH_AES_128_GCM_SHA256 = 0xC02D
+    ietfNames[0xC02D] = 'TLS_ECDH_ECDSA_WITH_AES_128_GCM_SHA256'
+    TLS_ECDH_ECDSA_WITH_AES_256_GCM_SHA384 = 0xC02E
+    ietfNames[0xC02E] = 'TLS_ECDH_ECDSA_WITH_AES_256_GCM_SHA384'
+
+    # RFC 5289 - ECC Ciphers with SHA-256/SHA-384 HMAC and AES-GCM
     TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 = 0xC02F
     ietfNames[0xC02F] = 'TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256'
     TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 = 0xC030
     ietfNames[0xC030] = 'TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384'
 
+    # RFC 5289 - ECC Ciphers with SHA-256/SHA-384 HMAC and AES-GCM
+    # unsupported
+    TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256 = 0xC031
+    ietfNames[0xC031] = 'TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256'
+    TLS_ECDH_RSA_WITH_AES_256_GCM_SHA384 = 0xC032
+    ietfNames[0xC032] = 'TLS_ECDH_RSA_WITH_AES_256_GCM_SHA384'
+
+    # draft-ietf-tls-chacha20-poly1305-00
+    # ChaCha20/Poly1305 based Cipher Suites for TLS1.2
+    TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_draft_00 = 0xCCA1
+    ietfNames[0xCCA1] = 'TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_draft_00'
+    TLS_DHE_RSA_WITH_CHACHA20_POLY1305_draft_00 = 0xCCA3
+    ietfNames[0xCCA3] = 'TLS_DHE_RSA_WITH_CHACHA20_POLY1305_draft_00'
+
+    # RFC 7905 - ChaCha20-Poly1305 Cipher Suites for TLS
+    TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256 = 0xCCA8
+    ietfNames[0xCCA8] = 'TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256'
+    TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256 = 0xCCAA
+    ietfNames[0xCCAA] = 'TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256'
+
 #pylint: enable = invalid-name
     #
     # Define cipher suite families below
     #
 
     # 3DES CBC ciphers
     tripleDESSuites = []
+    tripleDESSuites.append(TLS_ECDHE_ECDSA_WITH_3DES_EDE_CBC_SHA)  # unsupp
```

Comment:
As all the other comments, it should say the whole `unsupported` instead of `unsupp`. The difference can be confusing. Also appears in few other cases below.